src/util/accounts.ts

```diff
@@ -0,0 +1,8 @@
+export const storageMap: { [account: string]: number } = {
+  "none": 0,
+  "0x01": 0,
+  "0x02": 1,
+  "0x03": 2,
+  "0x04": 3,
+  "0x05": 4
+}
```

Comment:
Add new line at eof

Reply:
Changed in [this commit](https://github.com/onflow/flow-playground/pull/175/commits/289b5d04ba186213347bb27b87021fdfc8efaf3e)